benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -82,6 +87,7 @@ export default class Overview extends React.Component {
       filtered: [],
       tabIndex: 0,
       filterBoxVisible: false,
+      filter: getFilterParamsFromUrl(),
```

Comment:
This attribute `filter` is only initialized during the first rendering of the component but isn't used any further afterwards.